solr/core/src/java/org/apache/solr/pkg/PackageLoader.java

```diff
@@ -56,9 +52,12 @@
 public class PackageLoader implements Closeable {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
   public static final String LATEST = "$LATEST";
-  public static final String LOCAL_PACKAGES_DIR = System.getProperty("solr.packages.local.dir");
-  public static final String LOCAL_PACKAGES_WHITELIST = System.getProperty("solr.packages.local.whitelist", "");
+  public static final String LPD = "solr.packages.local.dir";
```

Comment:
Expand the acronym please.  I didn't know what it was at the spot that referenced it.